### DISCUSSION TITLE
I'm approved but the Extension says "User not authorized"

### DISCUSSION THREAD

**Discussion by 1rens1** [community/community#10666] [Copilot Conversations] [answered]
Hi, I recently got an email that I was approved. I quickly opened VSCode and the copilot says that I'm not authorized...
Thanks.
![image](https://user-images.githubusercontent.com/78131273/151477470-dc47ca03-831d-4af0-9426-44d47baf4f0a.png)

**Comment by 1rens1** [comment, marked as answer]
I sign out an sign in again.
That fixed the problem...

**Reply by bala418** [reply]
Where did you sign out from and sign in back?

**Reply by 1rens1** [reply]
> Where did you sign out from and sign in back?

In vscode

**Reply by MortPed** [reply]
How?

**Reply by MortPed** [reply]
In VSCODE, where is the option to signout (of whatever you signed out of)...

**Reply by MortPed** [reply]
> Where did you sign out from and sign in back?

https://github.com/github/feedback/discussions/6823
